pkgs/by-name/ar/artalk/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  artalk,
+  testers,
+}:
+buildGoModule rec {
+  pname = "artalk";
+  version = "2.8.5";
+
+  src = fetchFromGitHub {
+    owner = "ArtalkJS";
+    repo = "artalk";
+    rev = "v${version}";
+    hash = "sha256-pzjSig+vJ5QlpNE71UlC1MJdbvkIdswl+wiKMpjE/CM=";
+  };
+
+  CGO_ENABLED = 1;
+
+  vendorHash = "sha256-LI14Xfzd8AiDJq13H471MqocUn3jvkiXSuWeWOaSC40=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/ArtalkJS/Artalk/internal/config.Version=${version}"
+    "-X github.com/ArtalkJS/Artalk/internal/config.CommitHash=${src.outputHash}"
+  ];
+
+  passthru.tests = {
+    version = testers.testVersion {
+      package = artalk;
+      command = "${artalk}/bin/Artalk --version";
+    };
+  };
+
+  meta = with lib; {
```

Comment:
Please set `meta.mainProgram` to `"Artalk"`.
Among other things, this makes `testers.testVersion` run the correct command by itself, so you can remove the explicit command call.

Reply:
oh I forgot it
:(
Thx